src/components/navigation/Footer.tsx

```diff
@@ -42,15 +42,36 @@ const FooterLinks = ({ children, ...props }: FooterLinksProps) => {
   );
 };
 
-const FooterLink = ({ children, ...props }: LinkProps) => {
+interface FooterLinkProps extends LinkProps {
+  children: React.ReactNode;
+  linkComponent?: React.ElementType;
+  to?: string;
+  href?: string;
+}
+
+const FooterLink = ({
+  children,
+  linkComponent,
+  to,
+  href,
+  ...props
+}: FooterLinkProps) => {
   const theme = useTheme();
 
+  const shouldUseLinkComponent = linkComponent && to;
+
+  const linkProps = shouldUseLinkComponent
```

Comment:
Do we need this? Can't we just pass through what's been set on FooterLink? i.e. Get the user to decide what they want to use (and if they need "href" or "to").

Reply:
I think we just need to and linkComponent to the LinkProps (and href optional, if it isn't already)

Reply:
I could do it that way. I'd need to rename linkComponent to component to pass directly to Link. It would mean there's no fallback to href if the wrong props are used e.g. if component and href are passed together without to.

Reply:
It deleted my message...